src/components/file-viewer/examples/file-viewer.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, h } from '@stencil/core';
+/**
+ * Most common file types
+ *
+ * These are file formats that any web browser can display,
+ * without relying on any third-party plugins or additional
+ * plugins or extensions.
+ */
+@Component({
+    tag: 'limel-example-file-viewer',
+    shadow: true,
+    styleUrl: 'file-viewer.scss',
+})
+export class FileViewerExample {
+    private fileViewers = [
+        {
+            title: 'Image',
+            url: 'https://www.lime-technologies.se/wp-content/uploads/2021/02/SummerParty-8-scaled.jpg',
+            alt: 'Some of the people working at Lime Technologies',
+        },
+        {
+            title: 'Vector graphic',
+            url: 'https://lundalogik.github.io/lime-elements/versions/next/favicon.svg',
+            alt: 'Logo of Lime Elements',
+        },
+        {
+            title: 'PDF',
+            url: 'https://filesamples.com/samples/document/pdf/sample1.pdf',
```

Comment:
I really dislike fetching files form another domain for the demo. I'm gonna add some file samples amongst the files here. Do you have any suggestion where to upload such files? Can I add a folder here in the component @jgroth ? Or do you have a better idea? I also talked to our marketing folks. They are gonna remake and rebrand and we can't really host the files in Lime's website with peace of mind neither.

Reply:
What is it that you dislike? Is it fetching from another domain, or is it that we are not in control of those files? It's possible to include assets like images with the components, but you have to write code to get the correct URL for those files. 

If you just want to upload them somewhere and not worry too much about them, you can check out the `gh-pages` branch and add them there. They will then get an URL like `https://lundalogik.github.io/lime-elements/path/to/my-image.png`

Reply:
What I dislike is that these sites can disappear. We were using another site when we did the fist version of this document. They're now gone. Also we never know what's in the file. I don't know if there could be any security concerns about this neither. I was thinking it'd be nice to have Lime's own assets, both as a branding material for the company, and also for displaying nicer and more "real" file examples.

I'll check the branch you mentioned. should I put them here in a subfolder in the assets folder? 
<img width="326" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/2e4ab466-4bcf-4b7b-a3b4-1d2c54bc1d2b">